Foods.md

```diff
@@ -0,0 +1 @@
+-[ ] Tofu
```

Comment:
Need to list all the types of tofu
- `teriyaki tofu`
- `jerk tofu`
- buffalo tofu